create_infotexts.py

```diff
@@ -30,14 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commonscat", "wikidata"]
 
-    places_df.replace("-", np.nan)
+    places_df[places_df == "-"] = None
```

Comment:
Yes, good - the Pandas DataFrame is actually just intermediary to simplify the reading of Excel-files. Clarified in docstring.
Done.